micrometer-spring-legacy/src/main/java/io/micrometer/spring/MeterRegistryConfigurationSupport.java

```diff
@@ -41,6 +42,8 @@
 
     @PostConstruct
     void bindAll() {
+        MeterFactory.Config.setRegistry(registry);
```

Comment:
Here is how it gets wired in. However, I suspect it won't be as simple as I had hoped. I fear if a class had a static field or instance variables, then they would have already called the factory and retrieved their meters from the SimpleMeterRegistry that had been in place beforehand.

I'll play with it to determine if that is true. If so, I can make the default MeterRegistry account for it, but it will add complexity.